src/lib/crypto/kimchi/js/node_js/dune

```diff
@@ -22,13 +21,13 @@
   ../../wasm/Cargo.lock
   (source_tree ../../wasm/src)
   (source_tree ../../wasm/.cargo/config)
-  (source_tree ../../../proof-systems))
```

Comment:
Which commit did you use? Is it supposed to be part of the commit message "Update kimchi library with latest develop changes"?

Reply:
It seems to be reverted in https://github.com/MinaProtocol/mina/pull/14649/commits/4c9d45ce46afd65ff18dc5f3ff8648cd87c9c697.